packages/vue/src/components/molecules/SfGallery/SfGallery.vue

```diff
@@ -53,9 +53,11 @@
           @click="go(index)"
         >
           <SfImage
```

Comment:
@justyna-13 what do you think about change div from line 48 to button? You can focus on this element but can't select it

Reply:
and after that, you cant remove v-focus and tabindex from SfImage